src/index.js

```diff
@@ -210,6 +214,51 @@ class RuntimeMetrics {
     RuntimeMetrics.measureEvtLoopTime(this);
   }
 
+  static measureEvtLoopUtilization(self) {
+    const now = self.registry.hrtime();
+    const nanos = now[0] * 1e9 + now[1];
+    const lastNanos = self.lastEventLoopTime[0] * 1e9 + self.lastEventLoopTime[1];
+
+    const deltaNanos = nanos - lastNanos;
+    const last = self.lastEventLoop;
+    const current = self.eventLoopUtilization();
+    // compute idle percentage
+    const idle = current.idle * 1e6; // millis to nanos
+    const active = current.active * 1e6;
+    const deltaIdle = idle - last.idle * 1e6;
+    const deltaActive = active - last.active * 1e6;
+
+    self.eventLoopIdle.set(100.0 * deltaIdle / deltaNanos);
+    self.eventLoopActive.set(100.0 * deltaActive / deltaNanos);
+
+    self.lastEventLoopTime = now;
+    self.lastEventLoop = current;
+  }
+
+  _evtLoopUtilization() {
+    const reg = this.registry;
+
+    let eventLoopUtilization;
+    try {
+      eventLoopUtilization = require('perf_hooks').performance.eventLoopUtilization;
+    } catch (e) {
+      reg.logger.debug(`Got: ${e}`);
+    }
+
+    if (typeof eventLoopUtilization !== 'function') {
+      reg.logger.info(
```

Comment:
Should this be debug? Not sure how often the user would actually find it relevant.

Reply:
I chose info to avoid having to answer the question why isn't my app reporting this metric, and since it would only appear at startup it wouldn't be too inconvenient. I don't have a strong opinion either way.

Reply:
Fair enough.